Userland/Applications/Spreadsheet/SpreadsheetWidget.cpp

```diff
@@ -421,8 +421,7 @@ bool SpreadsheetWidget::request_close()
     if (!m_workbook->dirty())
         return true;
 
-    auto result = GUI::MessageBox::show(window(), "The spreadsheet has been modified. Would you like to save?", "Unsaved changes", GUI::MessageBox::Type::Warning, GUI::MessageBox::InputType::YesNoCancel);
-
+    auto result = GUI::MessageBox::ask_about_unsaved_changes(window(), current_filename());
```

Comment:
This commit isn't really "unifying" any popups (at least, when I read the commit title, I thought the commit would be unifying some duplicated behavior within the app itself).

I think better language would be "Use the system-wide unsaved changes dialog".